immutable-collection-encoding-test/pom.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.hubspot.immutables</groupId>
+    <artifactId>hubspot-immutables</artifactId>
+    <version>1.3-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>immutable-collection-encoding-test</artifactId>
```

Comment:
Why do we need a separate test module?

Reply:
the encoding annotations are themselves generated via an annotation processor and this makes it not possible to use them within the same module (the docs on the encodings also note that a separate module is necessary)